src/main/java/uk/gov/justice/digital/hmpps/keyworker/model/PrisonSupported.java

```diff
@@ -56,4 +56,20 @@ public boolean isMigrated() {
     public boolean hasPrisonersWithHighComplexityNeeds() {
         return hasPrisonersWithHighComplexityNeeds;
     }
+
+    public boolean isAutoAllocate() {
```

Comment:
Worth a chat on this boolean. I can't immediately see any usage of it in the API. I think it's only used in the UI where it is described as "Allow auto-allocation of prisoners to key workers". It means the buttons for auto allocate appear in the UI. If it is only used in the UI, might be best to rename it fully to `allowAutoAllocation` to show that it's not that it enables some background automatic allocation of people to key workers.

There's actually `private Boolean autoAllocationAllowed` in KeyworkerDto too which similarly looks like it may be a UI only value.